### PR TITLE
Background colours on page sections

### LINKED ISSUE #290: Amend templates/CSS to allow background colours on page sections

### PR DESCRIPTION
**Pull request by markconroy**
Closes #290

### CODE REVIEW

File: css/components/page-sections.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.lgd-page-section--bg-colour-1 {
+  background-color: var(--color-accent);
+}
+.lgd-page-section--bg-colour-2 {
+  background-color: var(--color-accent-alt);
```
Comment (msayoung) [inline css/components/page-sections.css]: I think we need to define --color-accent-alt in the variables.css file, however we could instead call this new variable --color-accent-2 to match what we've used in localgov_microsites_base

File: templates/content/node--localgov-news-article--full.html.twig [hunk 2]
```diff
@@ -83,36 +83,38 @@
...
-          <ul class="news-article__metadata-item news-article__metadata-item--categories" aria-label="{{ 'Article categories'|t }}">
-            {% for key,value in node.localgov_news_categories.value %}
-              <li class="news-article__category">
-                {{ content.localgov_news_categories[key] }}
-              </li>
-            {% endfor %}
-          </ul>
-        {% endif %}
-      </div>
+  <div class="lgd-container">
```
Comment (msayoung) [inline templates/content/node--localgov-news-article--full.html.twig]: do we want `padding-horizontal` class here too?

File: templates/layout/page.html.twig [hunk 3]
```diff
@@ -106,59 +106,73 @@
...
-        {# End Sidebars #}
+              {% else %}
+                {{ page.content }}
+            {% endif %}
+          {% endif %}
+          {# End Sidebars #}
 
-      </div>
-  </div>
+      {# </div>
```
Comment (msayoung) [inline templates/layout/page.html.twig]: these can be deleted.

File: templates/layout/page.html.twig [hunk 4]
```diff
@@ -106,59 +106,73 @@
     {{ page.content_top }}
   {% endif %}
 
-  <div class="lgd-container">
-      <div class="lgd-row">
+  {# <div class="lgd-container">
```
Comment (msayoung) [inline templates/layout/page.html.twig]: delete :)

File: css/components/page-sections.css (hunk 1 above)
Comment (markconroy) [reply css/components/page-sections.css]: Done.

File: templates/layout/page.html.twig (hunk 3 above)
Comment (markconroy) [reply templates/layout/page.html.twig]: Done.

File: templates/content/node--localgov-news-article--full.html.twig (hunk 2 above)
Comment (markconroy) [reply templates/content/node--localgov-news-article--full.html.twig]: Yes, I think so. Added now.

**Review by msayoung — APPROVED** [review]
I'm happy with this @markconroy . Great work :+1:

**Pull request merged**